.github/workflows/test-runner-resiliency-test.yml

```diff
@@ -0,0 +1,86 @@
+# This workflow will build and run a workspace manager resiliency test against the terra-wsmtest namespace
+
+name: Test Runner Resiliency Test
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 7 * * *' # run at 6 AM UTC, 2 AM ET.
+
+jobs:
+  test-runner-resiliency:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:13.1
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - name: Check out this repository
+        uses: actions/checkout@v2
+      - name: Set up AdoptOpenJDK 11
+        uses: joschi/setup-jdk@v2
+        with:
+          java-version: 11
+      # This step caches Gradle packages for reuse in the same VM instance.
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Get Vault token
+        id: vault-token-step
+        env:
+          VAULT_ADDR: https://clotho.broadinstitute.org:8200
+        run: |
+          VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
+            -e "VAULT_ADDR=${VAULT_ADDR}" \
+            vault:1.1.0 \
+            vault write -field token \
+              auth/approle/login role_id=${{ secrets.VAULT_APPROLE_ROLE_ID }} \
+              secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
+          echo ::set-output name=vault-token::$VAULT_TOKEN
+          echo ::add-mask::$VAULT_TOKEN
+
+      - name: Build the WSM client
+        run: |
+          echo "Building WSM client library"
+          ./gradlew :workspace-manager-client:clean :workspace-manager-client:assemble
+          cd workspace-manager-clienttests
+          export ORG_GRADLE_PROJECT_workspacemanagerclientjar=$(find .. -type f -name "workspace-manager-client*.jar")
+          echo "ORG_GRADLE_PROJECT_workspacemanagerclientjar = ${ORG_GRADLE_PROJECT_workspacemanagerclientjar}"
+
+      # We can also set the env var (TEST_RUNNER_SERVER_SPECIFICATION_FILE) in the GH action script to specify which server we are testing against.
```

Comment:
I think it would be clearer to just always set the server env var here. That way, if the default server ever changes in the suite/config file, the GH action won't break.

Reply:
Enforcing the use of `TEST_RUNNER_SERVER_SPECIFICATION_FILE` env var to override server specification in config file.